libindy/src/domain/wallet/mod.rs

```diff
@@ -42,5 +43,5 @@ pub struct ExportConfig {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Metadata {
```

Comment:
Seems we need to make metadata enum as content is different for different methods